colour/quality/cfi2017.py

```diff
@@ -93,7 +92,6 @@ class DataColorimetry_TCS_CIE2017:
 
     name: str
     XYZ: NDArrayFloat
-    CAM: CAM_Specification_CIECAM02
     JMh: NDArrayFloat
```

Comment:
Because I vecorized the cam call I have to I stack the values, then use the iterate operator to turn arrays into args for this data object constructor. It was around 5% or run time when I removed it. :/